pnpm-workspace.yaml

```
@@ -1,3 +1,2 @@
  packages:
-  - src/*
   - examples/*
```

to make the root package recognizable as a package for Changesets you should do this:
```suggestion
  - .
  - examples/*
```

Traditionally, we ignore root because it's rarely a versionable package